### ISSUE TITLE
GUD doesn't show indicator arrow in source file.

### ISSUE THREAD

**Issue by zarzen** [syl20bnr/spacemacs#5510]
#### Description
While using pdb or gdb in spacemacs, the indicator doesn't show at the current execution line.

#### Reproduction guide
- Start Emacs
- Open a python file
- `SPC :` -> `pdb`

*Observed behaviour:*
Spacemacs only show a cursor at the begin of the line which is been executing, like this:
![screen shot 2016-03-16 at 6 10 00 pm](https://cloud.githubusercontent.com/assets/1150493/13814296/a3eec988-ebc0-11e5-9c39-7857cf5d0cd6.png)

*Expected behaviour:*
The original gud mode works this way:
<img width="596" alt="screenshot 2016-03-16 15 50 38" src="https://cloud.githubusercontent.com/assets/1150493/13814199/4f8bbbb2-ebc0-11e5-8115-5d7d77b0568f.png">

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.14
- Spacemacs branch: master (rev. a298a5a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((c-c   :variables c-c  -enable-clang-support t)
 osx python java auto-completion better-defaults emacs-lisp git html org latex
 (shell :variables shell-default-shell 'eshell)
 spell-checking syntax-checking version-control)
```

**Comment by cpaulik** [comment]
This looks like a conflict with another package that spacemacs uses. I tried disabling `vi-tilde-fringe` and `flycheck` but it still does not show the arrow.

**Comment by StreakyCobra** [comment]
Random guesses: `linum`, `git-gutter[+]`?

**Comment by cpaulik** [comment]
I was wrong. Commenting out [these two lines](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+distribution/spacemacs-base/config.el#L213-L214) solves the problem.

@syl20bnr @TheBB Why is the `fringe-indicator-list` set to these values?

**Comment by TheBB** [comment]
@cpaulik Can you indicate the lines again? Maybe link to a fixed commit. :-)

**Comment by TheBB** [comment]
Okay I see, you meant `alist`, not `list`. I couldn't find it when searching.

I have no idea. Git blame points to a very old commit by @syl20bnr saying it makes Spacemacs "more starter kit friendly".

**Comment by syl20bnr** [comment]
wow this is a pre-spacemacs change and it should not be set in spacemacs, basically it disable the arrow and other indicators for wrapped lines and lines wider than the window. Somehow GUD uses these indicators.

Nice catch 👍

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
